provider/provider.go

```diff
@@ -52,12 +56,14 @@ func RunProvider(providerName, Version string) error {
 
 // Config struct holds the configuration fields for the provider
 type Config struct {
-	Mnemonic     string `pulumi:"mnemonic,optional"  provider:"secret"`
-	Network      string `pulumi:"network,optional"`
-	KeyType      string `pulumi:"key_type,optional"`
-	SubstrateURL string `pulumi:"substrate_url,optional"`
-	RelayURL     string `pulumi:"relay_url,optional"`
-	RmbTimeout   string `pulumi:"rmb_timeout,optional"`
+	Mnemonic          string  `pulumi:"mnemonic,optional"  provider:"secret"`
+	Network           string  `pulumi:"network,optional"`
+	KeyType           string  `pulumi:"key_type,optional"`
+	SubstrateURL      string  `pulumi:"substrate_url,optional"`
+	RelayURL          string  `pulumi:"relay_url,optional"`
+	RmbTimeout        string  `pulumi:"rmb_timeout,optional"`
+	Version           *string `pulumi:"version,optional"`
+	PluginDownloadURL *string `pulumi:"pluginDownloadURL,optional"`
```

Comment:
why did you add them?

Reply:
This solves #86, as [suggested](https://github.com/pulumi/pulumi-go-provider/issues/110#issuecomment-1696622421) by Pulumi devs.